backend/src/stwo/circuit_builder.rs

```diff
@@ -145,7 +174,7 @@ where
                     true => witness_eval[&poly_id][1].clone(),
                 },
                 PolynomialType::Constant => {
-                    unimplemented!("Constant polynomials are not supported in stwo yet")
+                    constant_eval[&poly_id].clone()
```

Comment:
Similar to committed polynomials, we want to be able to access the next row depending on `r.next`

Reply:
currently stwo only allows user to access pre-processed columns via get_preprocess_colums defined like this 
```
fn get_preprocessed_column(&mut self, _column: PreprocessedColumn) -> Self::F {
        let [mask_item] = self.next_interaction_mask(PREPROCESSED_TRACE_IDX, [0]);
        mask_item
    }
```
it is not able to pass offset, and if directly using next_interaction_mask(PREPROCESSED_TRACE_IDX, [0,1]) to get next reference for constant, it gives error "preprocessed columns can only be accessed by get_preprocessed_column"

Reply:
Uff, that's unfortunate. Let's focus on same row identities everywhere for now then.